platformio-tools.ini

```diff
@@ -65,6 +65,13 @@ build_flags =
   -DESP32C3
 board = esp32-c3-devkitm-1
 
+[env:BOARD_ES32C6DEVKITC1]
+platform = https://github.com/tasmota/platform-espressif32/releases/download/2024.04.14/platform-espressif32.zip
```

Comment:
Pls update as on main now. I belive that should work now.

Reply:
Unfortunately doesn´t work yet..
The ESP32C6 is supported in the esp32-arduino repo, but not in the platformio esp32 platform:
https://github.com/platformio/platform-espressif32/blob/develop/boards/esp32-c6-devkitc-1.json (arduino is missing from frameworks).

If i add it manually, at least my board doesn´t like it that much, i get errors like:

'''
E (120069) adc_cali: adc_cali_create_scheme_curve_fitting(79): Calibration required eFuse bits not burnt
[120001][E][esp32-hal-adc.c:322] __analogReadMilliVolts(): adc_cali_create_scheme_x failed!
'''

the tasmota repo works though...

but that might be related to me using Pre-Release Silicon which is missing the factory ADC calibration.